libcxx/test/std/utilities/format/format.functions/format_tests.h

```diff
@@ -3189,6 +3189,10 @@ void format_tests(TestFunction check, ExceptionTest check_exception) {
     const CharT* data = buffer;
     check(SV("hello 09azAZ!"), SV("hello {}"), data);
   }
+  {
```

Comment:
Can you leave a short comment behind? Maybe a link to the Github issue is sufficient.